package.json

```diff
@@ -104,19 +104,23 @@
         },
         {
             "path": "recipe/thirdpartyemailpassword/index.js",
-            "limit": "110kb"
+            "limit": "120kb"
         },
         {
             "path": "recipe/thirdparty/index.js",
-            "limit": "100kb"
+            "limit": "110kb"
```

Comment:
Why this increase?

Reply:
Hmm, I'm not sure why but it was failing for me locally. I wanted to check if there was an increase in 0.18 and these weren't updated before finalizing this PR. Running it again now, it seems to be OK with the old limits as well. I'm reverting these.